polkadot-parachains/statemine/src/xcm_config.rs

```diff
@@ -162,7 +162,7 @@ impl xcm_executor::Config for XcmConfig {
 	type LocationInverter = LocationInverter<Ancestry>;
 	type Barrier = Barrier;
 	type Weigher = FixedWeightBounds<UnitWeightCost, Call, MaxInstructions>;
-	type Trader = UsingComponents<IdentityFee<Balance>, KsmLocation, AccountId, Balances, ()>;
+	type Trader = UsingComponents<WeightToFee, KsmLocation, AccountId, Balances, ()>;
```

Comment:
One of the things that's still unresolved to me is where the extra weight should go when even the `refund_weight` method fails to refund all the surplus weight. Currently as we can see here, it is set to `()`, which means it all just gets dropped and forgotten, but in Polkadot, it gets refunded to the block author via the `ToAuthor` struct.

I see that in Cumulus, we have a `ToStakingPot` struct that sends nonzero balances to the staking pot, and perhaps this is what we should use?